### ISSUE TITLE
Search, return modifies file

### ISSUE THREAD

**Issue by flyfishMT** [jaredpar/VsVim#1562]
If I type /xxxx ENTER to search for "xxxx" it modifies (and checks out) the file. This happens in a typescript file (.ts) but not in a razor file (.cshtml).

**Comment by jaredpar** [comment]
Can you give me a bit more data here?  

- What version of Visual Studio are you using?
- How are you installing type script? 
- What source control provider do you have enabled?

**Comment by flyfishMT** [comment]
Hi,
VS 2013 Update 4
Installing the latest exe (1.4) from typscript website.
Using Team Foundation 2013. 

Let me know what else I can provide to help.

BTW - I have also noticed :w leaves an asterisk on the filename in the tab of a modified file, while CTL-S clears it.

**Comment by ckimes89** [comment]
I get this in VS 2015 in JS files and CSHTML files. I suspect there's some conflict with ReSharper since if my cursor is over a string in JS when I press enter (after finding something with `/`) then it splits the string onto a new line, which is a feature of ReSharper.

**Comment by jaredpar** [comment]
@ckimes89 I agree some interaction with ReSharper is likely at the heart of this problem.  Unfortunately I'm unable to reproduce it.  It's a real issue though as several users have reported this in various forms.  For whatever reason though it will not repro on any of my machinse (tried about 3 different setups now).  

I wonder if there is a simple ReSharper setting which causes this to repro / not repro.

**Comment by codernator** [comment]
@ckimes89 My group is experiencing the same issue, and has narrowed it down to Web Essentials. Resharper, as far as we can tell, is not relevant. We have suspended further digging because Web Essentials isn't for us.

**Comment by jaredpar** [comment]
@codernator interesting. I haven't tried that combination before.  Maybe that is the trick.
